var/spack/repos/builtin/packages/libsonata/package.py

```diff
@@ -34,7 +35,7 @@ class Libsonata(CMakePackage):
 
     def cmake_args(self):
         result = [
-            '-DEXTLIB_FROM_SUBMODULES=OFF',
+            '-DEXTLIB_FROM_SUBMODULES=ON',
```

Comment:
@wizmer : if we are using submodules now, could you check if all dependencies specified with `depends_on` are still required? (i.e. if py-pybind11 in being built as submodule, we can remove it from dependencies).

Reply:
Btw, if other things being equal, `depends_on` is preferred, can't we bump `pybind11` version in Spack recipe instead?

Reply:
We have been using latest release, there is no upstream release with the fixes that we need (if I am not mistaken).

Reply:
`pybind11@develop` or `pybind11@<commit>` until the necessary change become a part of the next `pybind11` release?